clarify expo updates sourcemaps for bare react-native app (#11612)
getsentry/sentry-docs: docs/platforms/react-native/sourcemaps/uploading/expo.mdx

To upload source maps for all platforms use the following command:

```bash
SENTRY_AUTH_TOKEN=___ORG_AUTH_TOKEN___ \
npx sentry-expo-upload-sourcemaps dist
```

To upload source maps without expo plugin (bare workflow):

```bash
SENTRY_AUTH_TOKEN=___ORG_AUTH_TOKEN___ \
SENTRY_PROJECT=___PROJECT_SLUG___ \
SENTRY_ORG=___ORG_SLUG___ \
SENTRY_URL=https://sentry.io/ \
npx sentry-expo-upload-sourcemaps dist
```

#### Notes

- `dist` is the default output directory of `eas update`.

> - Sentry Org, Sentry Project and Sentry Url are loaded from expo plugin config (app.json / app.config.js) if environment variable is not set.

Reviewer: ```suggestion
- Sentry Org, Sentry Project and Sentry Url will be loaded from expo plugin config (app.json / app.config.js) if environment variable is not set.
```

> - Sentry Org, Sentry Project and Sentry Url will be loaded from expo plugin config (app.json / app.config.js) if environment variable is not set.

Reviewer: ```suggestion
- Sentry Org, Sentry Project, and Sentry Url will be loaded from the expo plugin config (app.json / app.config.js) if the environment variable isn't set.
```
